### PR TITLE
New authors

### PR DESCRIPTION
**Pull request by thejessewinton**
## What does this PR do?

Add two new authors, Jesse Winton and Binyamin Yawitz.

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: static/images/avatars/jesse.png
Comment (loks0n) [inline static/images/avatars/jesse.png:1]: Should we downsize/scale ? Most images in the `avatars` folder are around 200kb

File: static/images/avatars/binyamin.png
Comment (loks0n) [inline static/images/avatars/binyamin.png:1]: Should this have a darker background? Maybe worth asking design to edit

File: src/routes/blog/author/binyamin-yawitz/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: author
+name: Binyamin Yawitz
+slug: binyamin-yawitz
+role: Software Eningeer
+bio: Developer with multilingual skills and systems experience
+github:   https://github.com/byawitz/
```
Comment (stnguyen90) [inline src/routes/blog/author/binyamin-yawitz/+page.markdoc:7]: missing avatar?

Comment (thejessewinton) [reply src/routes/blog/author/binyamin-yawitz/+page.markdoc:7]: good catch

File: static/images/avatars/jesse.png
Comment (thejessewinton) [reply static/images/avatars/jesse.png:1]: Will do.

File: static/images/avatars/binyamin.png
Comment (thejessewinton) [reply static/images/avatars/binyamin.png:1]: Might be; if it's not a blocker I can ping and have that done on Monday.

**Review by loks0n — APPROVED** [review]

**Pull request merged**